### PR TITLE
marua7665-week1

### PR DESCRIPTION
**Pull request by Sporky023**

### CODE REVIEW

File: deliverables/README.md [hunk 1]
```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
...
-For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
+**My Codecademy link for Html/CSS track completion is: http://www.codecademy.com/marua7665**
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+**What is the role of HTML in a web page?**  
+HTML provides the backbone structure to a web page. Most web pages can be thought of simply as
```
Comment (Sporky023) [inline deliverables/README.md:27]: well-put

File: deliverables/README.md [hunk 2]
```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
...
+**What are elements?**  
+Much like an atom that is made up of three sub-atomic particles consisting of protons, electrons and neutrons, 
+similarly, an HTML element is made up of a opening tag, closing tag, and text in between. Elements are
+the atomic building blocks of our HTML code.
+
+**What are tags?**  
+Tags are part of HTML elements that define the meaning and structure of our HTML code. They most often come
+in pairs, namely start and closing tags. Some tags may define a table within our code, some a heading,
+and we even define an html file using starting and closing html tags (<"html"></"html">).
+Sometimes tags can be used for styling (eg. <"strong"> [text] </"strong">, makes our [text] bold etc.), but
```
Comment (Sporky023) [inline deliverables/README.md:67]: To be precise, `<strong>` is an example of a semantic (i.e. meaningful) structure, intended to affect interpretation of the content, not just display.  

So for example, a screen reader for a blind person might alter its voice upon encountering a `<strong>` tag.  So it's like `<h1>`, `<h2>`, etc which not only have different default visual styles, but also indicate different roles for their content.  

Basically you can think of "semantic" stuff as things which would affect the flow of a machine which was reading the page.  Another example of a machine-based interpretation of a page could be Google's webcrawlers categorizing pages in order to display sub-links in search results.

File: deliverables/README.md [hunk 3]
```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
...
+for the most part, we leave the styling aspect to CSS.
+
+**What are attributes?**   
+HTML attributes are used to define different characteristics of elements. Attributes are entered within 
+opening tags, and they affect the behaviour of whatever is in between the opening and closing tags
+(eg. that can be a text, image etc). Most attributes have the following form: a type that initializes
```
Comment (Sporky023) [inline deliverables/README.md:73]: Some attributes have no value - their existence is itself the message to the browser.  For example  

`<input type="checkbox" checked />`  

This may not be 100% valid html5 syntax (or it could be, I don't know), but browsers will treat that the same as if you had put  

`<input type="checkbox" checked="checked" />`

File: deliverables/README.md [hunk 4]
```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
...
+a certain attribute (eg. "src" or "style"), followed by an equal sign, and then values in quotation marks.
+If we want to set our font to Arial, for example, we would do it as follows (can be any random opening and
+closing tag): <"p" "style="font-face:Arial""> [text] </"p"> (without the quotation marks, with the exception
+of the ones immediately after the equal sign.
+
+**What are forms?**   
+Forms are used to collect data from external source in the form of an input. This includes things like
```
Comment (Sporky023) [inline deliverables/README.md:80]: Great answer, but FYI a form also completes its purpose only with the `action` attribute in the opening tag, which specifies the URL which the browser should send a POST request to when "submit" is clicked.

File: deliverables/README.md [hunk 5]
```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
...
+
+**What are the three different ways to include CSS in your project or use CSS to style a particular element?**     
+First method is creating a separate CSS file which can be linked to the web page being built. Second method
+is creating a CSS block within the html code, most often being placed in between the opening and closing "head"
+tags. And the third method is insertion of the CSS code right in the html tag itself.
 
 #### Android
 Put your html code in the `android.html` file in this folder.
+Here is the link to my android preview:
+**http://htmlpreview.github.io/?https://github.com/marua7665/2015week1/blob/master/deliverables/android.html**
```
Comment (Sporky023) [inline deliverables/README.md:127]: His left arm doesn't move!  (his left, not ours)

File: deliverables/README.md [hunk 6]
```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
...
+SCM is a source control management system that allows for management of changes to documents or other large
+collections of information. SCM can be embedded in other systems to allow for fluid work with edits, making
+collaboration very easy.
+
+**What is a VCS?**                                                                         
+VCS are version control systems that behave similarly to SCM, but often run as stand-alone applications.
+
+**Why is Git useful for a developer?**                                                        
+For a developer, git is very useful because it saves different versions of the project being worked on. 
+Any change in a file is basically one version of that file, which the developer can access. This
```
Comment (Sporky023) [inline deliverables/README.md:145]: On thing to note is that files themselves are not individually versioned.  Each point in the commit tree refers to a state of the entire project as a whole.

File: deliverables/README.md [hunk 7]
```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
...
+
+**Why is Git useful for a developer?**                                                        
+For a developer, git is very useful because it saves different versions of the project being worked on. 
+Any change in a file is basically one version of that file, which the developer can access. This 
+also makes sharing the workload, as well as editing, very easy.
+
+**Why is Git useful for a team of developers?**                                             
+When a group of developers are working on the same project, fluidity in their collaboration is very 
+important. The fact that each of the developers are working on their own version of the project means that
+they can work on it and bring it back to the central file, with their changes accredited to their names.
```
Comment (Sporky023) [inline deliverables/README.md:151]: Also changes can be brought into the central repository in a controlled fashion.  So even if one feature was written in Febraury and another in March, the one written in March can be added first, or just specific commits from that feature branch can be cherry-picked into the main branch.

File: deliverables/README.md [hunk 8]
```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
...
+
+**How do you create a new Git repository for a project locally?**                                        
+A new git repository can be created using the command "git init" (without the quotation marks).
```
Comment (Sporky023) [inline deliverables/README.md:154]: To specify code, commands, variables, etc, you can surround the text with backticks (```).  Like `this`

File: deliverables/android.html [hunk 9]
```diff
@@ -3,20 +3,149 @@
...
+	-moz-transform:rotate(-30deg)translate(30px,0);
+    }
   </style>
 </head>
 <body>
   <div class='android'>
     <div class='head'>
-      <!-- put my head here -->
+        <div class='l_ant'></div>
+       	<div class='r_ant'></div>
```
Comment (Sporky023) [inline deliverables/android.html]: Be careful with indentation.  It's best to develop an obsession with whitespace at the beginning.  Like the fundamentals of a martial art, if you don't absolutely master it, then it will harm your efficiency for the rest of your career.  

If you see whitespace that's not correct (in this case only nesting should produce deeper indentation) you should do _nothing else_ until you've fixed the whitespace.
